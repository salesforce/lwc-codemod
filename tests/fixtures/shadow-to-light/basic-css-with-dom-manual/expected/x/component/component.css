:host {
    background: red;
}

.lwc-dom-manual-8a8lf9 div {
    color: blue;
}

.lwc-dom-manual-8a8lf9 span,.lwc-dom-manual-8a8lf9  button {
    font-size: 1.3rem;
}
