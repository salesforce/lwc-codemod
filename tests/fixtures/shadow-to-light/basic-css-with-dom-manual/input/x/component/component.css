:host {
    background: red;
}

div {
    color: blue;
}

span, button {
    font-size: 1.3rem;
}
