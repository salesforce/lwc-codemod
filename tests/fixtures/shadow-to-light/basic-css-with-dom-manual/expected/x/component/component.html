<template lwc:render-mode="light">
    <div class="lwc-dom-manual-8a8lf9"></div>
</template>
